### PR TITLE
Promote immutable secrets and configmaps to GA

### PR DESCRIPTION
**Pull request by wojtek-t**
Ref kubernetes/enhancements#1412

/milestone 1.21

### CODE REVIEW

**Comment by irvifa** [comment]
The PR to promote this to GA is already merged https://github.com/kubernetes/kubernetes/pull/97615

**Comment by kbhawkey** [comment]
Thanks @wojtek-t . Changes look good!
/lgtm
/cc @reylejano

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-26477--kubernetes-io-vnext-staging.netlify.app/docs/concepts/configuration/configmap/
https://deploy-preview-26477--kubernetes-io-vnext-staging.netlify.app/docs/concepts/configuration/secret/
https://deploy-preview-26477--kubernetes-io-vnext-staging.netlify.app/docs/reference/command-line-tools-reference/feature-gates/#feature-gates-for-alpha-or-beta-features

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -125,7 +125,8 @@ different Kubernetes components.
 | `HugePageStorageMediumSize` | `true` | Beta | 1.19 | |
 | `IPv6DualStack` | `false` | Alpha | 1.15 | |
 | `ImmutableEphemeralVolumes` | `false` | Alpha | 1.18 | 1.18 |
-| `ImmutableEphemeralVolumes` | `true` | Beta | 1.19 | |
+| `ImmutableEphemeralVolumes` | `true` | Beta | 1.19 | 1.20 |
```
Comment (kbhawkey) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:128]: Hi @wojtek-t .
Just noticed that this information should be in the "GA" table.
Would you move the three `ImmutableEphemeralVolumes` entries into the next table?

Comment (wojtek-t) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md]: done

**Review by wojtek-t — COMMENTED** [review]
@kbhawkey - PTAL

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md (hunk 1 above)
Comment (kbhawkey) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md]: Great!
/lgtm

**Pull request merged**